.travis.yml

```diff
@@ -8,6 +8,8 @@ python:
   - "pypy"
 # command to install dependencies
 install:
-- python setup.py install
+  - python setup.py install
 # command to run tests
-script: nosetests
\ No newline at end of file
+script:
+  - cd tests
+  - nosetests
```

Comment:
Nose is not part of the setup.py, why not just use setuptools with `test` command?

https://pythonhosted.org/setuptools/setuptools.html#test-build-package-and-run-a-unittest-suite